File: 9.Java/atendimento.java
public class atendimento {
    private int id;
    private String doenca;
    private int usuarioId;
    private String data;
    private String sintomas;

    public atendimento(String doenca, int usuarioId, String data, String sintomas) {
        this.doenca = doenca;
        this.usuarioId = usuarioId;
        this.data = data;
        this.sintomas = sintomas;
    }

    public int getId(){

        return id;
    }

    public void setId(int id){

        this.id = id;
    }

    public String getDoenca() {
        return doenca;
    }

    public void setDoenca(String doenca) {
        this.doenca = doenca;
    }

    public int getUsuarioId() {
        return usuarioId;
    }

    public void setUsuarioId(int usuarioId) {
        this.usuarioId = usuarioId;
    }

    public String getData() {
        return data;
    }

    public void setData(String data) {
        this.data = data;
    }

    public String getSintomas() {
        return sintomas;
    }

    public void setSintomas(String sintomas) {
        this.sintomas = sintomas;
    }

    @Override
    public String toString() {
        return "Atendimento - Doença: " + doenca + 
               ", Usuário ID: " + usuarioId + 
               ", Data: " + data + 
               ", Sintomas: " + sintomas;
    }
}

File: 4.model/atendimento.php
<?php

class atendimento {
    private $doenca;
    private $usuarioId;
    private $data;
    private $sintomas;

    public function getDoenca() {
        return $this->doenca;
    }

    public function setDoenca($doenca) {
        $this->doenca = $doenca;
    }

    public function getUsuarioId() {
        return $this->usuarioId;
    }

    public function setUsuarioId($usuarioId) {
        $this->usuarioId = $usuarioId;
    }

    public function getData() {
        return $this->data;
    }

    public function setData($data) {
        $this->data = $data;
    }

    public function getSintomas() {
        return $this->sintomas;
    }

    public function setSintomas($sintomas) {
        $this->sintomas = $sintomas;
    }

    public function __toString() {
        return "Atendimento - Doença: {$this->doenca}, Usuário ID: {$this->usuarioId}, Data: {$this->data}, Sintomas: {$this->sintomas}";
    }
}

?>
